### PR TITLE
feat: add mobile: stopMediaProjectionRecording uploadTimeout

### PR DESCRIPTION
**Pull request by mohanjith**
New argument uploadTimeout added to mobile: stopMediaProjectionRecording documentation.

For: https://github.com/appium/appium/issues/18549

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -994,6 +994,7 @@ method | string | no | The http multipart upload method name. The 'PUT' one is u
 headers | Map&lt;string, string&gt; | no | Additional headers mapping for multipart http(s) uploads | {'Agent': '007'}
 fileFieldName | string | no | The name of the form field, where the file content BLOB should be stored for http(s) uploads. `file` by default | blob
 formFields | Map&lt;string, string&gt; or Array&lt;Pair&gt; | no | Additional form fields for multipart http(s) uploads. | {'name': 'yolo.mp4'}
+uploadTimeout | number | no | The maximum number of milliseconds to wait until the media file is uploaded to the remote location. 60000 ms by default. | 30000
```
Comment (mykola-mokhnach) [inline README.md:997]: haven't we bumped it to 5 minutes by default?

**Comment by mykola-mokhnach** [comment]
Please also bump the minimum version of appium-android-driver in package.json

**Comment by KazuCocoa** [comment]
https://github.com/appium/appium-uiautomator2-driver/blob/master/package.json#L61 should be `5.10.2`

**Review by ankit-gautam23 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## [2.18.0](https://github.com/appium/appium-uiautomator2-driver/compare/v2.17.0...v2.18.0) (2023-04-20)
```
Comment (KazuCocoa) [inline CHANGELOG.md:1]: this change is not needed since a release action will do this automatically by checking the commit message

Comment (KazuCocoa) [reply CHANGELOG.md:1]: oh, sorry. this is current master's one

**Pull request closed without merging**